### PR TITLE
Add command that will output the JDK version information

### PR DESCRIPTION
**Pull request by dusanboskovic**
This command will be able to retrieve the version info of a JDK's core file 
The output should be equivalent to running `java -version` on the command line

Signed-off-by: Dusan-Boskovic <dusan.boskovic@ibm.com>

### CODE REVIEW

**Comment by dusanboskovic** [comment]
@keithc-ca 
This is the command that I will be using to perform the IBM/OpenJ9 test

**Comment by dusanboskovic** [comment]
This addition is based off of @manqingl's code

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/VersionInfoCommand.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	}
+	
+	public void run(String command, String[] args, Context context, PrintStream out)
+			throws DDRInteractiveCommandException 
+	{
+		initialize();
+		Object javaFullVersion = properties.get("java.fullversion");
+		Object javaRuntimeVersion = properties.get("java.runtime.version");
+		Object javaRuntimeName = properties.get("java.runtime.name");
+		if (javaRuntimeName == null) {
```
Comment (keithc-ca) [inline debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/VersionInfoCommand.java:48]: When I tried this, both `javaRuntimeName` and `javaRuntimeVersion` were `null`: we need to understand why those system properties are not available.

**Comment by keithc-ca** [comment]
I came across the `!coreinfo` command which seems to provide the information presented by this command, including the `java.runtime.version` and much more.

**Comment by dusanboskovic** [comment]
> I came across the `!coreinfo` command which seems to provide the information presented by this command, including the `java.runtime.version` and much more.

The `!coreinfo` command does not provide the "java.vm.name" property, which should either Eclipse OpenJ9 VM or IBM J9 VM.
This is the output on an OpenJ9 build
```
>!coreinfo
...
JAVA SERVICE LEVEL INFO
JRE 1.8.0 Linux amd64-64 (build 1.8.0_242-b08)
JAVA VERSION INFO
JRE 1.8.0 Linux amd64-64-Bit Compressed References 20200122_511 (JIT enabled, AOT enabled)
OpenJ9   - 51a5857d2
OMR      - 7a1b0239a
JCL      - 8cf8a30581 based on jdk8u242-b08
JAVA VM VERSION	- openj9-0.18.1
```
vs. the output for `versioninfo`
```
> versioninfo
Eclipse OpenJ9 VM(JRE 1.8.0 Linux amd64-64-Bit Compressed References 20200122_511 (JIT enabled, AOT enabled)
OpenJ9   - 51a5857d2
OMR      - 7a1b0239a
JCL      - 8cf8a30581 based on jdk8u242-b08)
```

I was thinking, maybe instead of creating a new command, I can just add this feature from `versioninfo` to `!coreinfo` instead?

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/VersionInfoCommand.java (hunk 1 above)
Comment (dusanboskovic) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/VersionInfoCommand.java:48]: > When I tried this, both `javaRuntimeName` and `javaRuntimeVersion` were `null`: we need to understand why those system properties are not available.

I am unsure why this is the case. I tested on a couple of different builds for both IBM and OpenJ9 and I saw the expected output

**Comment by pshipton** [comment]
> I was thinking, maybe instead of creating a new command, I can just add this feature from versioninfo to !coreinfo instead?

I think this is the better approach.

**Comment by dusanboskovic** [comment]
I will be opening a new PR with that adds "java.vm.name" property to the `!coreinfo` command

**Pull request closed without merging**